extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -704,7 +703,7 @@ export class VisibilityManagerForEmbed extends VisibilityManager {
 
   /** @override */
   getRootLayoutBox() {
-    const rootElement = dev().assertElement(this.embed.host);
```

Comment:
Looks like we are still using `embed.host` as root to determine `rootVisibility`, but we are using `embed.iframe` as root to calculate `rootLayoutBox`. It's a bit confusing, please make sure you add document it clearly : )

Reply:
Ok, can do. Good catch!

Reply:
Done